ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/extensions/PaywallDataExtensions.kt

```diff
@@ -0,0 +1,102 @@
+package com.revenuecat.purchases.ui.revenuecatui.extensions
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.platform.LocalContext
+import androidx.core.os.LocaleListCompat
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.paywalls.PaywallColor
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.data.processed.PaywallTemplate
+import java.net.URL
+
+/***
+ * Default [PaywallData] to display when attempting to present a [PaywallView] with an offering that has no paywall
+ * configuration, or when that configuration is invalid.
+ */
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.Companion.createDefault(packages: List<Package>): PaywallData {
+    return PaywallData.createDefaultForIdentifiers(packages.map { it.identifier })
+}
+
+@Composable
+@ReadOnlyComposable
+private fun PaywallData.Companion.createDefaultForIdentifiers(packageIdentifiers: List<String>): PaywallData {
+    val context = LocalContext.current
+    val locale = LocaleListCompat.getDefault()[0].toString()
```

Comment:
This is duplicated in `PaywallData.localizedConfiguration`. Maybe we can add a property there to make sure we use the same default?

Reply:
I did this https://github.com/RevenueCat/purchases-android/pull/1261#discussion_r1334327448 so this is not needed anymore